_includes/parse-server/push-notifications-android.md

```diff
@@ -1,67 +1,48 @@
-#### Configure Broadcast Receiver and Permissions
+#### FCM Push Setup
 
-Add the following service and broadcast receiver definitions to `AndroidManifest.xml` immediately before the *closing* `</application>` tag:
+Add dependency to the application level `build.gradle` file.
 
-```xml
-<service android:name="com.parse.PushService" />
-<receiver android:name="com.parse.ParsePushBroadcastReceiver"
-android:exported="false">
-<intent-filter>
-<action android:name="com.parse.push.intent.RECEIVE" />
-<action android:name="com.parse.push.intent.DELETE" />
-<action android:name="com.parse.push.intent.OPEN" />
-</intent-filter>
-</receiver>
-<receiver android:name="com.parse.GcmBroadcastReceiver"
-android:permission="com.google.android.c2dm.permission.SEND">
-<intent-filter>
-<action android:name="com.google.android.c2dm.intent.RECEIVE" />
-<action android:name="com.google.android.c2dm.intent.REGISTRATION" />
-
-<!--
-IMPORTANT: Change "com.parse.starter" to match your app's package name.
--->
-<category android:name="com.parse.starter" />
-</intent-filter>
-</receiver>
-
-<!--
-IMPORTANT: Change "YOUR_SENDER_ID" to your GCM Sender Id.
--->
-<meta-data android:name="com.parse.push.gcm_sender_id"
-  android:value="id:YOUR_SENDER_ID" />;
+```groovy
+dependencies {
+    implementation 'com.parse:parse-android-fcm:latest.version.here'
+}
 ```
+Then, follow Google's docs for [setting up an Firebase app](https://firebase.google.com/docs/android/setup). Although the steps are different for setting up FCM with Parse, it is also a good idea to read over the [Firebase FCM Setup](https://firebase.google.com/docs/cloud-messaging/android/client).
 
-Change the `android:name` attribute of `<category>` element above to match your application's package name.
```

Comment:
We also should mention the migration from GCM to FCM link (https://developers.google.com/cloud-messaging/android/android-migrate-fcm)

Reply:
I think we could potentially mention it in the README for GCM, but not here, since hopefully most won't need to care about it.